pages/common/todoist.md

```diff
@@ -0,0 +1,28 @@
+# todoist
+
+> This program will let you use Todoist from the CLI.
+> More information: <https://github.com/sachaos/todoist>.
+
+- Add a task:
+
+`todoist add "{{some_task}}"`
+
+- Add a high priority task, with label, project, and due date:
+
+`todoist add "{{some_task}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
```

Comment:
```suggestion
`todoist add "{{task_name}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
```

for the date, can "{{tomorrow 9am}}" work?

Reply:
yes, that would work too, the todoist api uses natural language